utils/R/read.output.R

```diff
@@ -123,8 +123,6 @@ read.output <- function(runid, outdir, start.year=NA,
 
   if(!nofiles){
     for(ncfile in ncfiles) {
-      valid.nc <- tryl(nc_open(ncfile))
```

Comment:
@robkooper that doesn't completely solve the problem, because if the try fails to open the file we have no way of calling nc_close. The list of valid files is coming from searching the directory, so the file DOES exists. If it's not valid netCDF are we better off failing completely or contributing to an open file leak?